deploy/charts/arcadia/templates/ingress-minio.yaml

```diff
@@ -0,0 +1,77 @@
+{{- if .Values.minio.ingress.api.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}-minio-api-ingress
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    description: minio-api-ingress
+    displayName: minio-api-ingress
+    httpSend: /
+    ingress-lb: portal-ingress
+    kubernetes.io/ingress.class: portal-ingress
+    nginx.ingress.kubernetes.io/enable-access-log: "false"
+    nginx.ingress.kubernetes.io/enable-rewrite-log: "false"
+    nginx.ingress.kubernetes.io/load-balance: round_robin
+    nginx.ingress.kubernetes.io/proxy-body-size: ""
```

Comment:
Maybe set `nginx.ingress.kubernetes.io/proxy-body-size` to `1024m`

Reply:
We need to upload large files in portal

Reply:
increased to 10240m = 10G